libimage/runtime.go

```diff
@@ -161,8 +161,14 @@ type LookupImageOptions struct {
 
 	// If set, do not look for items/instances in the manifest list that
 	// match the current platform but return the manifest list as is.
+	// only check for manifest list, return ErrNotAManifestList if not found.
 	lookupManifest bool
 
+	// If matching images resolves to a manifest list, return manifest list
+	// instead of resolving to image instance, if manifest list is not found
+	// try resolving image.
+	ReturnManifestIfPresent bool
```

Comment:
Sorry for the late request: Could we rename the option to just `ManifestList`? `LookupImageOptions{ManifestList: true}` seems quite intuitive and much shorter.

Reply:
@vrothberg sure can do that, should we make it `ManifestListIfPresent` ? Since it returns manifest list if present otherwise returns image

Reply:
I prefer without `IfPresent`.  It feels intuitive that it would resolve to a manifest list only if it's a manifest list.  The comment goes into the details in case users wonder.

Reply:
@vrothberg cool I agree, we can always refer to comments.